rwr/player.py

```diff
@@ -73,7 +73,6 @@ def craft(cls, rwr_account, rwr_account_stat):
 
         ret.username = rwr_account.username.encode('iso-8859-1').decode('utf-8')
 
-        ret.leaderboard_position = rwr_account_stat.leaderboard_position
```

Comment:
Make sure it's not displayed in the discord command because it's no longer available in stats history